odl/trafos/fourier.py

```diff
@@ -1365,13 +1366,25 @@ def _call_pyfftw(self, x, out, **kwargs):
         return out
 
     @property
+    def adjoint(self):
+        """The adjoint Fourier transform."""
+        if self.domain.exponent != 2.0 or self.range.exponent != 2.0:
+            raise NotImplementedError(
+                'Adjoint only implemented for exponent=2')
```

Comment:
My suggestion is to leave this as is and perhaps make a TODO and an issue with this exact discussion quoted. In fact, i just did this: #741.